### DISCUSSION TITLE
had to go to beta 5, beta 6 didnt work

### DISCUSSION THREAD

**Discussion by emerymastering** [apohl79/audiogridder#759] [Help]
Hello Andreas,

I tried beta 6 today and connection didn't work, went back to beta 5 and all works fine.
If I opened a session in Ableton or Nuendo then I was able to load one instance and add a plugin with beta 6.
But opening a session with 2 Agridders with many plugins started freezing Nuendo and I could not see any plugin loaded.
Felt like it tried to connect but something doesn't work.
I am on osx 10.15.7 used VST3 version

**Comment by apohl79** [comment]
If possible, please provide the beta6 logs of server and plugin. Thx

**Reply by emerymastering** [reply]
[server logs.zip](https://github.com/apohl79/audiogridder/files/7338740/server.logs.zip)
[plugin logs.zip](https://github.com/apohl79/audiogridder/files/7338743/plugin.logs.zip)
Attaching the logs:)

**Reply by apohl79** [reply]
Thanks for the logs. Will take a closer look.

You could try sandboxing. It might help stability.

**Reply by apohl79** [reply]
Also, just to confirm something, can you turn off logging and tracing in the plugin and the server and see if that works? 

I added some debugging to parameter syncing to track down an automation bug, and I see a ton of these messages in your logs. So that might actually kill performance.

**Reply by emerymastering** [reply]
How do i try sandboxing?
Ok will install v6 again and disable logging and tracing.

**Reply by apohl79** [reply]
You can enable sandboxing in the server settings window at the upper end. Thx!

**Reply by emerymastering** [reply]
So After testing and reading another threat I see that it might be related to changed Juice plugin ids.
Coz loading fresh and inserting plugs works fine. But If I try to load preset or older project then it doesn't work. It loads names of plugs but they don't show up.
So it means I have to redo all the presets again right?
It's it really needed to update to v6? Will it make some substantial improvements? Or sticking to v5 is fine as well?

**Reply by apohl79** [reply]
I'm trying to confirm this right now. If that's the issue, I need to recall beta6 and see what changes went into JUCE and either roll back or patch JUCE.

**Reply by emerymastering** [reply]
cool:)

**Reply by apohl79** [reply]
I think the ID generation is NOT broken. In your case, it might be the logging. Did it make a difference, if you disabled logging and tracing?

> It's it really needed to update to v6? Will it make some substantial improvements? Or sticking to v5 is fine as well?

beta6 is mostly adding bug fixes. If you had no issues, all good. For me beta6 is much more stable and I can load sessions with sanboxing without any plugin crashes on the server side most of the time.

**Reply by emerymastering** [reply]
Hey, Disabling logging and tracing gave no effect, was not able to open old session or presets. Only able to create new ones.
So felt like mismatched IDs indeed or something else if IDs atay same?
